Easydict/NewApp/Feature/Shortcut/Shortcut.swift

```diff
@@ -163,9 +163,9 @@ extension Shortcut {
 extension Shortcut {
     func bindingShortcut(keyCombo: KeyCombo?, type: ShortcutType) {
         guard let keyCombo else {
-            HotKeyCenter.shared.unregisterHotKey(with: type.rawValue)
```

Comment:
When deleting the shortcut key, keyCombo is nil. If unregisterHotKey is not executed, the shortcut key will still be effective.